src/updater.cpp

```diff
@@ -484,7 +484,7 @@ namespace appimage::update {
 
             std::ostringstream oss;
             oss << "'" << gpgPath << "' "
-            << "--no-default-keyring --keyring '" << tempKeyRingPath << "' --import";
+            << "--no-default-keyring --keyring '" << tempKeyRingPath << "' --import 2>/dev/null";
```

Comment:
You should use `--logger-file /dev/null --status-file /dev/nulll`, I guess. This will work in the unlikely case that we're ever going to replace `popen` there.

Reply:
I still get output with those options. Looks like it doesn't even matter whether there is an issue with the signature or not.

Reply:
Doesn't seem to work. Your solution works, though.